scripts/generate-component-config/build-configs.js

```diff
@@ -0,0 +1,18 @@
+const terraNameSpacedRepos = [
+  'terra-core',
```

Comment:
Should terra-clinical be added to this list?

Reply:
No it doesn't need to be. terra-core and terra-framework components are named 'terra-component-name' where as terra-clinical's are name-spaced 'terra-clinical-component-name'.